sbt-plugin/src/sbt-test/gen-java/03-no-java-package/build.sbt

```diff
@@ -1,4 +1,5 @@
 resolvers += Resolver.sonatypeRepo("staging")
+resolvers += Resolver.bintrayRepo("akka", "snapshots")
```

Comment:
Maybe the tests can later be refactored to use a linked base `base-settings.sbt`? So, things like that can be more easily changed?

Reply:
you mean like a symlink? IIRC those are problematic on Windows...